src/custom/state/index.ts

```diff
@@ -41,9 +42,10 @@ const reducers = {
   orders,
   price,
   gas,
+  userMod, // contains user state. Created to not modify the already existing "user" state inherited from uniswap
```

Comment:
What about using a different name for the state then?

Like `announcements` or something else :shrug:

Reply:
It's meant to be for more properties, this is  just the first one. I will call it profile in case u like it more

Reply:
https://github.com/gnosis/cowswap/pull/1200/commits/36227f7e11419e334252c3af8a4468935a41a86c

Reply:
I'm not against profile, it sounds good, but... we are adding a page with that name.
I'm afraid this might be confusing...